src/modules/debug/status.ts

```diff
@@ -6,9 +6,9 @@ import getMajorSemver from 'semver/functions/major.js'
 
 // Following lines bellow are automatically updated with GitHub Action when Bee version is updated
 // so if you are changing anything about them change the `update_bee` action accordingly!
-export const SUPPORTED_BEE_VERSION_EXACT = '1.4.3-1213e063'
-export const SUPPORTED_API_VERSION = '2.0.0'
-export const SUPPORTED_DEBUG_API_VERSION = '1.2.1'
+export const SUPPORTED_BEE_VERSION_EXACT = '1.5.0-rc3-89021656'
+export const SUPPORTED_API_VERSION = ' 3.0.0'
+export const SUPPORTED_DEBUG_API_VERSION = ' 2.0.0'
```

Comment:
These extra spaces should also be removed.
```suggestion
export const SUPPORTED_API_VERSION = '3.0.0'
export const SUPPORTED_DEBUG_API_VERSION = '2.0.0'
```